src/replica/Task.h

```diff
@@ -200,46 +179,37 @@ class Task : public EventLogger, public std::enable_shared_from_this<Task> {
      */
     virtual void onStop() {}
 
-    /// @return a flag indicating if the task needs to be stopped
+    /// @return A flag indicating if the task needs to be stopped.
     bool stopRequested() const { return _stopRequested.load(); }
 
-    /**
-     * @return the context string to be used when logging messages into
-     * a log stream.
-     */
+    /// @return The context string to be used when logging messages into a log stream.
     std::string context() const;
 
     /**
-     * Log a message into the Logger's LOG_LVL_INFO stream
-     *
-     * @param msg
-     *   a message to be logged
+     * Log a message into the Logger's LOG_LVL_INFO stream.
+     * @param msg A message to be logged
      */
     void info(std::string const& msg) { LOGS(_log, LOG_LVL_INFO, context() << msg); }
```

Comment:
This is an unconventional use of logging, I think you are supposed to use `LOGS` macro in the code directly, otherwise the context information in the log message is not very useful.

Reply:
I will think about this. The method was introduced to reduce the noise in the code by calling:
```
info("Something happened");
```
vs:
```
LOGS(_log, LOG_LVL_INFO, context() << "Something happened");
```
I will investigate an option to replace `info()` with the macro-definition to benefit from both.
This pattern is presently used in all modules of the R-I system's REST API, not just here. So, for the sake of code consistency, I'd rather do this as a separate refactoring effort rather than fixing this particular usage.
(I maintain a list of things to refactor in Confluence which is presently down).